components/usage/pkg/db/workspace_instance_usage.go

```diff
@@ -49,9 +49,9 @@ func CreateUsageRecords(ctx context.Context, conn *gorm.DB, records []WorkspaceI
 	return db.CreateInBatches(records, 1000).Error
 }
 
-type order int
+type Order int
```

Comment:
Had to make it public, as I needed to assign a variable in the calling code in `api` and without it, I'd have needed more duplication.